Tella/Components/Connections/ReportsPageView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ReportsPageView.swift
```

Comment:
same here the reference of this calss is still in the project if you can remove it